src/components/01-branding/02-typography/paragraph/paragraph-default.config.yml

```diff
@@ -0,0 +1,13 @@
+label: Paragraph
+status: wip
+preview: "@sam"
+variants:
+  - name: default
```

Comment:
Please remove default variant. It will automatically be provided as the standard variant.